src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -2290,6 +2290,49 @@ sdap_nested_group_deref_send(TALLOC_CTX *mem_ctx,
     return req;
 }
 
+static hash_table_t *
+convert_ldb_element_to_set(const struct ldb_message_element *members)
+{
+    errno_t ret;
+    hash_table_t *set = NULL;
+    hash_key_t key;
+    hash_value_t value;
+    size_t j;
+
+    ret = sss_hash_create(NULL, 0, &set);
```

Comment:
Since you've analyzed the issues with unsuitable `count` options I wonder if it would have any benefits to use `members->num_values` here?